src/misc/lv_types.h

```diff
@@ -55,6 +58,8 @@ extern "C" {
  *      TYPEDEFS
  **********************/
 
+#ifndef __ASSEMBLY__
```

Comment:
For future reference please add comment here why it is required and what problem it solves.

Reply:
Done